MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2285,7 +2285,7 @@ protected void loadOptionsFile() {
             String xn = wn.getNodeName();
 
             if (xn.equalsIgnoreCase("campaignOptions")) {
-                options = CampaignOptions.generateCampaignOptionsFromXml(wn);
+                options = CampaignOptions.generateCampaignOptionsFromXml(wn, version);
```

Comment:
What's with the addition of the 'version' parameter?

Reply:
It was required for a migration issue, but I swapped it to use a check because the alternative value is not valid for MRMS.
However, I think it would make sense to include the version in the options xml itself because of presets.